packages/botonic-react/src/constants.js

```diff
@@ -0,0 +1,36 @@
+export const CUSTOM_WEBCHAT_PROPERTIES = {
+  brandColor: 'brand.color',
```

Comment:
It would be safer to use the usage of these properties CUSTOM_WEBCHAT_PROPERTIES. brandColor instead of 'brand.color', but I understand it would make the code more verbose.
At least maybe we could find a way to ensure during runtime that we don't pass a bad property name

Reply:
i write it down for next iterations 👍